### PR TITLE
Geo-location from IP address

### PR DESCRIPTION
**Pull request by Namyalg**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #106 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/Location-based-on-IP/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Geo-location based on IP address
+- This script can be used to obtain the latitude and longitude from where a particular IP address originates
+- The API from https://ip-api.com/ has been used for the same
+
+## Usage
+- Requirements
```
Comment (tejan-singh) [inline Python/Location-based-on-IP/README.md:6]: please specify commands to install requirements.

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: Python/Location-based-on-IP/README.md (hunk 1 above)
Comment (Namyalg) [reply Python/Location-based-on-IP/README.md:6]: Done

**Comment by Namyalg** [comment]
Can I raise issues to automate sending messages in telegram and to build a telegram bot in Python?

**Review by tejan-singh — CHANGES_REQUESTED** [review]
Capture4

requirements missing. please update readme.

**Comment by tejan-singh** [comment]
@Namyalg Yes, you can raise the issue but check if the similar issue already opened or merged.

**Comment by Namyalg** [comment]
@tejan-singh , my script runs with pip install requests, I have updated that in the README

**Comment by tejan-singh** [comment]
Capture

More requirements needed.

**Comment by tejan-singh** [comment]
Which operating system are you testing this script on ? Windows or Linux

**Comment by Namyalg** [comment]
I have tried on both linux and Windows

Namya LG

**Comment by tejan-singh** [comment]
Okey, I will talk to other mentors for review.

**Pull request merged**